Datasets/data-connect-dataset-calendarview.md

```diff
@@ -49,38 +49,38 @@ The CalendarView_v0 dataset can be joined with relevant events, users, and other
+| location | object | Location information of the event. *Format:* `STRUCT<DisplayName: STRING, Address: STRUCT<Street: STRING, City: STRING, State: STRING, CountryOrRegion: STRING, PostalCode: STRING>, Coordinates: STRUCT<Altitude: DOUBLE, Latitude: DOUBLE, Longitude: DOUBLE, Accuracy: DOUBLE, AltitudeAccuracy: DOUBLE>>.` | No | None |
 | isAllDay | boolean | True if the event lasts all day. Adjusting this property requires adjusting the Start and End properties of the event as well; otherwise false. | No | None |
 | isCancelled | boolean | True if the event has been canceled; otherwise false. | No | None |
 | isOrganizer | boolean | True if the message sender is also the organizer; otherwise false. | No | None |
-| onlineMeeting | string | Details for an attendee to join the meeting online. | No | None |
+| onlineMeeting | object | Details for an attendee to join the meeting online. | No | None |
 | onlineMeetingProvider | string | Represents the online meeting service provider. *Possible values:* teamsForBusiness, vskypeForBusiness, and skypeForConsumer. | No | None |
-| recurrence | string | The recurrence pattern for the event. *Format:* `STRUCT<Pattern: STRUCT<Type: STRING,`Interval`: INT, Month: INT, DayOfMonth: INT, DaysOfWeek: ARRAY<STRING>, FirstDayOfWeek: STRING, Index: STRING>,`Range`: STRUCT<Type: STRING, StartDate: STRING, EnDate: STRING, RecurrenceTimeZone: STRING, NumberOfOccurrences: INT>>.` | No | None |
+| recurrence | object | The recurrence pattern for the event. *Format:* `STRUCT<Pattern: STRUCT<Type: STRING,`Interval`: INT, Month: INT, DayOfMonth: INT, DaysOfWeek: ARRAY<STRING>, FirstDayOfWeek: STRING, Index: STRING>,`Range`: STRUCT<Type: STRING, StartDate: STRING, EnDate: STRING, RecurrenceTimeZone: STRING, NumberOfOccurrences: INT>>.` | No | None |
 | responseRequested | boolean | True if the sender would like a response when the event is accepted or declined; otherwise false. | No | None |
 | showAs | string | The status to show. *Possible values:* free, tentative, busy, oof, workingelsewhere, unknown. | No | None |
 | transactionId | string | A custom identifier specified by a client app for the server to avoid redundant POST operations in case of client retries to create the same event.  | No | None |
 | type | string | The event type. *Possible values:* singleinstance, occurrence, exception, seriesmaster. | No | None |
-| attendees | string | The collection of attendees for the event. *Format:* `ARRAY<STRUCT<EmailAddress: STRUCT<Name: STRING, Address: STRING>, Status: STRUCT<Response: STRING, Time: STRING>, Type: STRING>>.` | No | None |
+| attendees | array | The collection of attendees for the event. *Format:* `ARRAY<STRUCT<EmailAddress: STRUCT<Name: STRING, Address: STRING>, Status: STRUCT<Response: STRING, Time: STRING>, Type: STRING>>.` | No | None |
 | organizer | string | The organizer of the event. *Format:* `STRUCT<EmailAddress: STRUCT<Name: STRING, Address: STRING>>.` | No | None |
```

Comment:
for organizer, type will be object, instead of string

Reply:
Fixed it